### ISSUE TITLE
Vlna package missing a small 'a' letter.

### ISSUE THREAD

**Issue by vojta-horanek** [olsak/OpTeX#178]
Is there a reason why the vlna package only [defines](https://github.com/olsak/OpTeX/blob/fed3c68f27d63c4b1e596809af5ca2b112bb3c76/optex/pkg/vlna.opm#L150) single chars for Czech like this?
```tex
\singlechars{Czech}{AIiVvOoUuSsZzKk}
```

Shouldn't it also [include the small](https://prirucka.ujc.cas.cz/?id=880) 'a' letter? Or am I missing something?

**Comment by olsak** [comment]
Opinions about the Czech conjunction "a" (and) ... its prohibition to be at the end of line ... differ. One argument is that it is syllabic word, so it can be at the end of line. Moreover, it is conjunction, it binds the text before it with the same power as the text after it, so there is no reason to prohibit it at the end of the line and allow it at the beginning of next line. I came to this opinion during the discussion among typographers. It was very long time ago, before the norm mentioned [here](https://github.com/michal-h21/luavlna/issues/13) began to exist. And I was among the correctors of this norm and I was mentioned that "A" (at beginning of the sentence) can be prohibited but "a" can't. My remark was not accepted. The argument: to be more simple. But my old books and defaults for my old external program "vlna" keeps my opinion about "a" conjunction and doesn't respect the norm. I copied this default to the defaults of `vlna.opm` because I mean that this is better regardless of a norm. Everybody can re-define \singlechars if it is needed.
